data/pirate jobs.txt

```diff
@@ -2581,6 +2581,362 @@ mission "Cargo Theft (South)"
 		outfit "Secret Cargo" -1
 		payment 1500000
 		dialog "The pirates of <planet> take the secret cargo from your cargo hold and pay you <payment>."
+		
+mission "FW Assasination [1]"
+	name "Assassinate a political target"
+	description "A key polititican is out on a pleasure flight in their ship, the <npc>. Find them and destroy them."
+	repeat
+	job
+	to offer
+		"combat rating" > 200
+	source
+		government "Pirate"
+		attributes "south pirate"
+	npc kill
+		government "Free Worlds"
+		personality heroic staying target
+		system
+			distance 1 3
+		fleet
+			names civilian
+			variant
+				"Flivver"
+		dialog "<npc> has been destroyed, and it's pilot killed. Return to <destination> for you payment."
```

Comment:
`<npc> has been destroyed, and it's pilot killed.` seems redundant. 

for your* payment